tests/unit/jobs/ContentJobsTest.php

```diff
@@ -737,4 +737,24 @@ public function itShouldForceDeleteOneContent()
         $this->assertNull(Content::find($first->id));
         $this->assertNotNull(Content::withTrashed()->find($second->id));
     }
+
+    /** @test */
+    public function cantCreateContentWithNonExistingThumb()
```

Comment:
Test name is about creating content and you are updating it, so you should use create content job, or the name of this test changed to cant update.